packages/api/cms-api/src/blob-storage/backends/s3/blob-storage-s3.storage.ts

```diff
@@ -12,7 +12,11 @@ export class BlobStorageS3Storage implements BlobStorageBackendInterface {
 
     constructor(config: BlobStorageS3Config["s3"]) {
         this.client = new AWS.S3({
-            requestHandler: config.requestHandler,
+            requestHandler: config.requestHandler ?? {
+                // https://github.com/aws/aws-sdk-js-v3/blob/main/supplemental-docs/CLIENTS.md#request-handler-requesthandler
+                requestTimeout: 60000, // clean up unused connections after 60 seconds
```

Comment:
I doubt this comment is correct

Reply:
Yes it is. That's the only reason I'm setting this timeout. If no request/response was sent via a connection for 60 seconds, that indicates the file stream was terminated -> the connection is blocked but not used by anyone anymore. The timeout ensures that these unused connections are cleaned up.

How would you phrase that?

Reply:
> That's the only reason I'm setting this timeout. If no request/response was sent via a connection for 60 seconds, that indicates the file stream was terminated -> the connection is blocked but not used by anyone anymore. The timeout ensures that these unused connections are cleaned up.

Maybe exactly that? Or something like:

```ts
// Workaround to prevent socket exhaustion caused by dangling streams (e.g., when the user leaves the site).
// Close the connection when no request/response was sent for 60 seconds, indicating that the file stream was terminated.
```

Reply:
Sounds good cfbaef5b2df7a896b00e3d821e21f6fcb979db33